pkgs/development/compilers/gcc/6/default.nix

```diff
@@ -155,13 +155,13 @@ let version = "6.4.0";
           "--enable-threads=win32"
           "--enable-sjlj-exceptions"
           "--enable-hash-synchronization"
-          "--disable-libssp"
+          "--enable-libssp"
           "--disable-nls"
           "--with-dwarf2"
           # I think noone uses shared gcc libs in mingw, so we better do the same.
           # In any case, mingw32 g++ linking is broken by default with shared libs,
           # unless adding "-lsupc++" to any linking command. I don't know why.
```

Comment:
So this old comment about mingw32 g++ linking being broken with shared libs must be wrong.

Reply:
@DavidEGrayson or we didn't build any C++ lol.